packages/nodes-base/nodes/Ftp/Ftp.node.ts

```diff
@@ -256,6 +256,29 @@ export class Ftp implements INodeType {
 				description: 'The new path',
 				required: true,
 			},
+			{
+				displayName: 'Options',
+				name: 'options',
+				type: 'collection',
+				placeholder: 'Add Field',
+				default: {},
+				displayOptions: {
+					show: {
+						operation: [
+							'rename',
+						],
+					},
+				},
+				options: [
+					{
+						displayName: 'Move',
+						name: 'move',
+						type: 'boolean',
+						default: false,
+						description: 'Whether to also move the file when the "New Path" uses a different directory than the "Old Path"',
```

Comment:
Hey @RicardoE105!

Thanks for the addition. Could we change the option a little to more closely match the action performed? Perhaps something like the below? 😀

`displayName: 'Create Directories on Rename'`
`description: 'Recursively create destination directory when renaming an existing file or folder'`